Jakefile

```diff
@@ -161,17 +161,30 @@ function generateChromeManifest() {
   });
 }
 
+/**
+ * Exports the files to a target, used to export into mozilla-central
+ *
+ */
+function exportFiles(target) {
+  let mcLocation = getMozillaCentralLocation();
+  let extTargetDir = path.join(mcLocation, target);
+  jake.rmRf(extTargetDir);
+  jake.cpR(BUILD_DIR, extTargetDir);
+
+  console.log(`Exported built sources into ${extTargetDir}`);
+}
+
 desc(`Builds the extension into the ${BUILD_DIR}/ directory`);
 task("build", ["building:cleanup", "building:copy"], () => {});
 
 desc("Exports the sources into mozilla-central");
 task("export-mc", ["build"], () => {
-  let mcLocation = getMozillaCentralLocation();
-  let extTargetDir = path.join(mcLocation, "browser/extensions/webcompat");
-  jake.rmRf(extTargetDir);
-  jake.cpR(BUILD_DIR, extTargetDir);
+  exportFiles("browser/extensions/webcompat");
+});
 
-  console.log(`Exported built sources into ${extTargetDir}`);
+desc("Exports the sources into the mozilla-central for android");
+task("export-mc-android", ["build"], () => {
```

Comment:
So we only have one extension for both desktop and android.

What I think is we only need to export-mc, which export file to **browser/extensions/webcompat'** and **mobile/android/extensions/webcompat** no matter what change we make, to prevent we don't miss any change to commit to m-c. But I don't know if other reviewer will have concern to put same source code in 2 different folders.